Source/WebCore/rendering/RenderTableCol.cpp

```diff
@@ -67,21 +69,23 @@ RenderTableCol::~RenderTableCol() = default;
 void RenderTableCol::styleDidChange(StyleDifference diff, const RenderStyle* oldStyle)
 {
     RenderBox::styleDidChange(diff, oldStyle);
-    RenderTable* table = this->table();
+    CheckedPtr table = this->table();
     if (!table)
         return;
     // If border was changed, notify table.
     if (!oldStyle)
         return;
-    table->invalidateCollapsedBordersAfterStyleChangeIfNeeded(*oldStyle, style());
+
+    CheckedRef<const RenderStyle> newStyle = style();
```

Comment:
`CheckedRef newStyle` is fine here. We don't typically specify if something is const